src/main/java/org/jabref/JabRefMain.java

```diff
@@ -65,6 +69,25 @@ private static void start(String[] args) {
         }
 
         Globals.prefs = preferences;
+
+        // Check if we are running an acceptable version of Java
+        final BuildInfo buildInfo = Globals.BUILD_INFO;
```

Comment:
I would extract this new code at least to a new method.

Reply:
A new method of `JabRefMain` or a different place?

Reply:
In my opinion, the code can stay in `JabRefMain`.